xmipy/xmiwrapper.py

```diff
@@ -499,6 +512,14 @@ def set_value(self, name: str, values: NDArray[Any]) -> None:
                 c_char_p(name.encode()),
                 byref(values.ctypes.data_as(POINTER(c_int))),
             )
+        elif var_type_lower.startswith("logical"):
```

Comment:
Seems like that was missed in the modflow 6 review.
Variable types in BMI should follow numpy [dtypes](https://numpy.org/doc/stable/reference/arrays.dtypes.html).
So this should be `bool` not `logical`.

Reply:
A counter argument is that the numpy dtypes are not exactly followed in xmipy, e.g. STRING vs.  bytes, FLOAT vs float32, DOUBLE vs float64, INTEGER vs int32 or int64 (depending on what `get_var_itemsize` returns either 4 or 8).

The statement could be rephrased to match either:
```python
        elif var_type_lower.startswith("logical") or var_type_lower.startswith("bool")
```

Reply:
I would change some of these names too if I would do it over again, but string, float, double and integer are valid numpy types, logical is not. I really think that should be changed on the MODFLOW side :)